force-app/main/default/staticresources/pmm/quickActionModal.css

```diff
@@ -0,0 +1,4 @@
+.slds-modal__container {
```

Comment:
Scoping could be tricky - the component itself is nested a few layers WITHIN this container, so it may require some less-conventional css at least. Do you have that trick up your sleeve or should I go searching? 

Are these directives any stickier than the equivalent ones declared in the addMoreParticipants aura component?

Reply:
Take a look at bsdt overrides

Reply:
Switch to Aura no longer an issue.